contracts/Portfolio.sol

```diff
@@ -733,11 +738,11 @@ abstract contract PortfolioVirtual is Objective {
             = FVM.decodeSwap(data);
             _swap(args);
         } else if (instruction == FVM.ALLOCATE) {
-            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity) =
+            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity, uint128 maxDeltaAsset, uint128 maxDeltaQuote) =
                 FVM.decodeAllocateOrDeallocate(data);
-            _allocate(useMax == 1, poolId, deltaLiquidity);
+            _allocate(useMax == 1, poolId, deltaLiquidity, maxDeltaAsset, maxDeltaQuote);
         } else if (instruction == FVM.DEALLOCATE) {
-            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity) =
+            (uint8 useMax, uint64 poolId, uint128 deltaLiquidity, uint128 minDeltaAsset, uint128 minDeltaQuote) =
                 FVM.decodeAllocateOrDeallocate(data);
             _deallocate(useMax == 1, poolId, deltaLiquidity);
```

Comment:
is this GitHub display bug? For me, it looks like you're calling `_deallocate(useMax == 1, poolId, deltaLiquidity);` without the min amounts which should be impossible because you change `_deallocate` to take the two additional parameters.

Reply:
Good catch, looks like it was not a display bug; fixed now: https://github.com/primitivefinance/portfolio/commit/cbce5a751b8db005c31fce0664c6f5f36c8a2fdf

Reply:
I noticed that yesterday while merging but then fixed it on the `spearbit/post-audit` branch, I just don't know what happened, it looks like resolving the conflicts resulted in a weird merge...